src/main/java/org/truffleruby/language/exceptions/RescueNodeWrapper.java

```diff
@@ -50,6 +50,7 @@ public NodeCost getCost() {
             return NodeCost.NONE;
         }
 
+        @SuppressWarnings("deprecation")
```

Comment:
What is deprecated in this method?

Reply:
`onReturnExceptional` - there's a new pattern for it. I will follow this PR with the new pattern to use it, but I thought it was best to update Truffle in an isolated update. Especially now it's failing CI!